packages/strapi-admin/admin/src/translations/en.json

```diff
@@ -342,5 +342,6 @@
   "notification.success.saved": "Saved",
   "notification.version.update.message": "A new version of Strapi is available!",
   "notification.version.update.link": "See more",
-  "request.error.model.unknown": "This model doesn't exist"
+  "request.error.model.unknown": "This model doesn't exist",
+  "request.error.entry.duplicate": "Duplicate entry"
```

Comment:
For now can you only return the error as english text and not use translation keys. We will provider error codes instead in the future as a first version of this just returning an english emssage will be good enough & avoid maintenance in the front end when we change this